Feature1/create to do item

Link-Spirit opened the pull request:
Initialized Main Menu, allowing user selection of creating task (Title and Optional Date) and selection of 'exit'

Link-Spirit committed:
Struct Fix.

Link-Spirit committed:
.

Link-Spirit committed:
.

Link-Spirit committed:
Intialize () , re-formated and while loop exit condition modified.

Link-Spirit committed:
TodoManger. GetAllTodoItems()

Relocated to the bottom line 68

jacieylyn commented on TaskManager_ConsoleApp_Terry/App.cs:
```diff
@@ -0,0 +1,93 @@
+
+        public void Intialize()
+        {
+            Console.WriteLine($"Type '{Selection.Create}' to create a task");
```
I feel like creating a whole class to just hold strings that are only re-used once is not that essential. I think you should just replace these references with strings or (if you really want to organize it in its own section) create a struct holding all the strings.

JCruz6725 replied:
That's funny, I thought the complete opposite. I like the single location for constants. I'm unsure of the implications of using a struct versus a class in this case.

jacieylyn reviewed (commented):
Just a suggestion about the selection class reference in the App.cs

Muhammadou1 commented on TaskManager_ConsoleApp_Terry/App.cs:
```diff
@@ -0,0 +1,93 @@
+                        break;
+                }
+
+                if (run) { 
```
Maybe there is a reason for it but i think the if statement here is not needed. you can call the GetAllTodo inside of the switch and then render the todo

JCruz6725 commented on TaskManager_ConsoleApp_Terry/App.cs:
```diff
@@ -0,0 +1,93 @@
+        public void Intialize()
+        {
+            Console.WriteLine($"Type '{Selection.Create}' to create a task");
+            Console.WriteLine($"Type '{Selection.Update}' to update a task");
```
Let's move this into the task render class and have the app call that function, as outlined in the design document.

JCruz6725 commented on TaskManager_ConsoleApp_Terry/Program.cs:
```diff
@@ -1,10 +1,21 @@
+            App App = new App();
+            App.Intialize();
+            App.Main();
+            //App.Shutdown();
```
Re-enable this code.

JCruz6725 reviewed (changes requested):
Needs work

JCruz6725 commented:
One more thing. The console should clear/refresh it-self between menus.

Link-Spirit committed:
1.Intialize() has been cleared out and now Render.TaskViewer handles the function when Intialize() calls the function .
2.Enabled App.Shutdown.
3.Removed Empty un needed spaces
4.Console.Clear() have been implemented allowing console to look much less cluttered between menus and inputs

JCruz6725 reviewed (approved):
Will need some clean up. Good to go.
